example/java/build_library.sh

```diff
@@ -1,8 +1,8 @@
-#!/bin/bash
+#!/bin/bash -e
 cd "$(dirname "$0")"
 
 # folder with bindings and native library
-mkdir dist
+mkdir dist || true
```

Comment:
Useful PR.
Please, consider `mkdir -p` option instead `|| true` before merge.

Reply:
how is that better than this?

Reply:
`mkdir -p` creates dir if it doesn't exist and will not return fail in case it is exists, so there is no need to `|| true`

Reply:
also `mkdir -p` fails in case it is actually impossible to create dir while `mkdir dist || true` will continue script